### DISCUSSION TITLE
Primary / Secondary Inputs

### DISCUSSION THREAD

**Discussion by k5beason** [forkineye/ESPixelStick#564] [General]
Hello,

I would like to use FPP Remote as the primary input and MQTT as a Secondary. Running 4.0 Beta 4 there is no option for such a setup.

Ideally I'd like to retain MQTT control through Home Assistant with he built in effects but  be able to send data via FPP remote as well.

**Comment by MartinMueller2003** [comment]
This would be an enhancement request. At the moment, FPP and MQTT are at a peer level. There would be no way way to know when to use one mode vs the other.

**Reply by k5beason** [reply]
It was an option in 4.0 Beta 3. Would it be possible to have it an option on both Primary and Secondary? giving us an option if the xlights bug is the lesser of two evils

**Comment by MartinMueller2003** [comment]
You can put this in as an enhancement request.
